corehq/apps/hqwebapp/static/hqwebapp/js/inactivity.js

```diff
@@ -66,9 +66,12 @@ hqDefine('hqwebapp/js/inactivity', [
                 success: function (data) {
                     if (!data.success) {
                         var $body = $modal.find(".modal-body");
+                        var src = initialPageData.reverse('iframe_login');
+                        src += "?next=" + initialPageData.reverse('iframe_login_new_window');
+                        src += "&username=" + initialPageData.get('secure_timeout_username');
```

Comment:
This is already in the template?

Reply:
yes, it's already being used in this file, in `pollToHideModal`